script.module.clouddrive.common/resources/skins/default/1080i/export-main-dialog.xml

```diff
@@ -0,0 +1,356 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
similarly, skinning stuff is functionality of a script extension point...not a script.module

Reply:
Same as before. All the addons needs these custom windows. Why repeat and copy the xmls in all the addons when this 'module' can abstract them.